index.js

```diff
@@ -114,9 +126,11 @@ Use the getFlavorByIndex function below to do the following:
     For example: running getFlavorByIndex(originalFlavors, 2) would return "Black Walnut", assuming Rainbow Sherbert has been added successfully
 */
 
-function getFlavorByIndex(/*your code here*/){
-    /*your code here*/
+function getFlavorByIndex(originalFlavors, a){
```

Comment:
Consider changing function parameter variable name a to index or indexNumber.  This just conveys more meaning and makes your code easier to read.